docs/getting_started/sso/azure.md

```diff
@@ -0,0 +1,29 @@
+# SEKOIA.IO: Configure Single Sign-on with Azure Active Directory
+
+In order to configure Okta with SEKOIA.IO, the following steps must be done:
+
+1. Create a new application on Okta.
```

Comment:
Are we talking about Okta or Azure here?

Reply:
:facepalm: fixed